### PR TITLE
Add a few missing things following 3D release

### PR DESCRIPTION
**Pull request by AlexandreSi**
* Restore Debug draw action in games with 3D
* Fix inspector not working on 3D objects with Z = 0
* Use Magenta box when no model is specified for model 3D objects
* Add missing default names for 2 new 3D objects

### CODE REVIEW

File: newIDE/app/src/ObjectsList/index.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ const objectTypeToDefaultName = {
   'MyDummyExtension::DummyObject': 'NewDummyObject',
   'Lighting::LightObject': 'NewLight',
   'TextInput::TextInputObject': 'NewTextInput',
+  'Scene3D::Model3DObject': 'NewModel3D',
```
Comment (ClementPasteau) [inline newIDE/app/src/ObjectsList/index.js:75]: do we have restrictions on the name or can we use `New3DModel` here if this is just user-facing?
It's not consistent with the code, but I think it's more english

**Review by ClementPasteau — COMMENTED** [review]
lgtm!

**Review by ClementPasteau — APPROVED** [review]
approving to you can merge if you're confident

File: GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts [hunk 2]
```diff
@@ -218,6 +219,11 @@ namespace gdjs {
 
         // Uncomment to display some debug metrics from Three.js.
         // console.log(threeRenderer.info);
+
+        const debugContainer = this._runtimeScene
+          .getDebuggerRenderer()
+          .getRendererObject();
+        if (debugContainer) pixiRenderer.render(debugContainer);
```
Comment (4ian) [inline GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts:226]: You're doing a Pixi render, so you must ensure the state is ready for it, like done for layers.

This should be guarded like it's done for a 2D only rendering with 
```if (lastRenderWas3D) {
                    // Ensure the state is clean for PixiJS to render.
                    threeRenderer.resetState();
                    pixiRenderer.reset();
                  }
```

and set `lastRenderWas3D = false`
and done before the `if (!lastRenderWas3D)`

**Pull request merged**